rules/linux/privilege_escalation_suspicious_cap_setuid_python_execution.toml

```
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2023/09/05"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2023/09/05"
+
+[rule]
+author = ["Elastic"]
+description = """
+This detection rule monitors for the execution of a system command with setuid or setgid capabilities via Python, 
+followed by a uid or guid change to the root user. This sequence of events may indicate successful privilege escalation.
```

```suggestion
followed by a uid or gid change to the root user. This sequence of events may indicate successful privilege escalation.
```
Idk, but gid seems to be a better fit in the context